src/Persistence.RavenDB/RavenDBViewModelHelper.cs

```diff
@@ -0,0 +1,23 @@
+﻿// <copyright file="RavenDBViewModelHelper.cs" company="Cognisant">
+// Copyright (c) Cognisant. All rights reserved.
+// </copyright>
+
+namespace CR.ViewModels.Persistence.RavenDB
+{
+    /// <summary>
+    /// Helper class used for code shared between <see cref="RavenDBViewModelReader"/> and <see cref="RavenDBViewModelWriter"/>
+    /// </summary>
+    internal static class RavenDBViewModelHelper
+    {
+        /// <summary>
+        /// Helper method used to generate the ID of the RavenDB document that a view model of type TEntity with specified key should be stored.
+        /// </summary>
+        /// <typeparam name="TEntity">The type of the View Model</typeparam>
+        /// <param name="key">The key of the view model</param>
+        /// <returns>The ID of the RavenDB document.</returns>
+        internal static string MakeID<TEntity>(string key)
+        {
+            return typeof(TEntity).FullName + "/" + key;
```

Comment:
String interpolation would be a _little_ nicer, especially with an expression bodied property.